src/org/thoughtcrime/securesms/components/emoji/EmojiProvider.java

```diff
@@ -158,6 +172,26 @@ public int getIntrinsicHeight() {
       this.intrinsicHeight = EMOJI_RAW_HEIGHT * decodeScale;
     }
 
+    private Bitmap getEmojiBitmap(float scale) {
+      Bitmap singleEmoji = Bitmap.createBitmap((int) intrinsicWidth/2, (int) intrinsicHeight/2, Bitmap.Config.ARGB_8888);
```

Comment:
Shouldn't this be `intrinsicWidth * scale`? Probably makes sense to try that setting different scale (`2.5f`, `0.7f`) actually works.

Reply:
+1 nice catch

Reply:
fixed and tested with different scales